src/common/proto-irc.c

```diff
@@ -1165,13 +1165,14 @@ process_named_msg (session *sess, char *type, char *word[], char *word_eol[],
 		case WORDL('I','N','V','I'):
 			if (ignore_check (word[1], IG_INVI))
 				return;
-			
-			if (word[4][0] == ':')
-				EMIT_SIGNAL_TIMESTAMP (XP_TE_INVITED, sess, word[4] + 1, nick,
-											  serv->servername, NULL, 0,
+
+			text = STRIP_COLON(word, word_eol, 4);
+			if (g_strcmp0 (word[3], serv->nick))
```

Comment:
I assume this should be case-insensitive. Use `serv->p_cmp()` (not `NULL` safe).

Reply:
Fixed.